code/game/machinery/computer/_computer.dm

```diff
@@ -90,6 +90,16 @@
 		playsound(loc, 'sound/effects/glassbr3.ogg', 100, TRUE)
 		set_light(0)
 
+/obj/machinery/computer/proc/imprint_gps(gps_tag) // Currently used by the upload computers and communications console
+	var/tracker = gps_tag
+	if(!tracker) // Don't give a null GPS signal if there is none
+		return
+	for(var/obj/item/circuitboard/computer/board in src.contents)
```

Comment:
This should just access `circuit` instead of doing a contents search

Reply:
doing so breaks the PR (I just tested it)